### ISSUE TITLE
Failed to create JSON object.

### ISSUE THREAD

**Issue by shuukaidoyuzuli** [itdelatrisu/opsu#140]
**Version:** 0.10.1
**Build date:** 2015-07-19 04:39
**OS:** Linux (i386)
**JRE:** 1.7.0_51
**Error:** `Failed to create JSON object.`
**Stack trace:**
```
org.json.JSONException: A JSONObject text must begin with '{' at 1 [character 2 line 1]
	at org.json.JSONTokener.syntaxError(JSONTokener.java:433)
	at org.json.JSONObject.<init>(JSONObject.java:194)
	at org.json.JSONObject.<init>(JSONObject.java:321)
	at itdelatrisu.opsu.Utils.readJsonObjectFromUrl(Utils.java:560)
	at itdelatrisu.opsu.downloads.servers.BloodcatServer.resultList(BloodcatServer.java:71)
	at itdelatrisu.opsu.states.DownloadsMenu$1.run(DownloadsMenu.java:409)
```

**Comment by kirito587** [comment]
Yeah , i have the same problem , how to fix it?

**Comment by itdelatrisu** [comment]
Same problem as #139, fixed in 9c7dbce.  Thank you!
